### DISCUSSION TITLE
{{CURRENT_WEEKDAY}} is not working

### DISCUSSION THREAD

**Discussion by walker-chen2024** [open-webui/open-webui#7372] [Issues]
{{CURRENT_WEEKDAY}} is not working, we added "CURRENT_WEEKDAY:{{CURRENT_WEEKDAY}}" in the system prompt word test, and the output result did not change at all.
Not expected ex:CURRENT_WEEKDAY:Monday

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/c8a592f2-5416-4b92-940b-ac7299f6911f)

Unable to reproduce, keep us updated!

**Reply by walker-chen2024** [reply]
I use vllm and OpenWebUI. The vllm logs help me understand the data input into the LLM. As shown in the attached picture, I included {{CURRENT_TIME}} and {{CURRENT_WEEKDAY}} in the system prompt and asked the question "what weekday is today" Then, I observed the vllm logs. As shown in the picture, {{CURRENT_TIME}} was successfully converted, but {{CURRENT_WEEKDAY}} was not.
![openwebui-1](https://github.com/user-attachments/assets/37b98136-6e04-4045-8966-5c4b911596a0)
![openwebui-2](https://github.com/user-attachments/assets/8c3113f1-4443-4682-bbcd-456c893ab713)

**Comment by Jefferderp** [comment]
I'm experiencing a similar issue. These variables are not expanding:
![image](https://github.com/user-attachments/assets/cf7494e7-ffa1-4c10-91e1-c4d0c6f7ed6e)
